frontend/src/components/ProfileContent.tsx

```diff
@@ -1,28 +1,44 @@
 import { useMsal } from '@azure/msal-react';
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { requestMSAuthResult } from '../functions/requestMSAuthResult';
+import { Alert, AlertTitle } from '@mui/material';
 
 export const ProfileContent = () => {
   const { accounts, instance } = useMsal();
   const name = accounts[0] && accounts[0].name;
+  const isValidEmail =
+    /^[A-Za-z0-9._%+-]+@(unsw.edu.au|unswalumni.com|ad.unsw.edu.au|zmail.unsw.edu.au|student.unsw.edu.au)$/.test(
+      accounts[0].username
+    );
```

Comment:
Rather than having this so high up in the component, it makes more sense to have it in the fetchData function as we will _only_ be using it there.